lib/couch_tap/changes.rb

```diff
@@ -18,6 +18,7 @@ def initialize(opts = '', &block)
       @source   = CouchRest.database(opts)
       @http     = HTTPClient.new
       @http.debug_dev = STDOUT if ENV.key?('DEBUG')
+      @skip_seqs = Set.new
```

Comment:
Nice to learn something new: http://ruby-doc.org/stdlib-2.4.1/libdoc/set/rdoc/Set.html